pwndbg/ida.py

```diff
@@ -76,10 +87,20 @@ def init_ida_rpc_client():
 
     if exception:
         if not isinstance(_ida_last_exception, exception[0]) or _ida_last_exception.args != exception[1].args:
-            print(message.error("[!] Ida Pro xmlrpc error"))
-            traceback.print_exception(*exception)
+            if hasattr(pwndbg.config, "exception_verbose") and pwndbg.config.exception_verbose:
+                print(message.error("[!] Ida Pro xmlrpc error"))
+                traceback.print_exception(*exception)
+            else:
+                exc_type, exc_value, _ = exception
+                print(message.error('Failed to connect to IDA Pro ({}: {})'.format(exc_type.__qualname__, exc_value)))
+                if exc_type is socket.timeout:
+                    print(message.notice('To increase the time to wait for IDA Pro use `') + message.hint('set ida-timeout <new-timeout>') + message.notice('`'))
```

Comment:
We should probably say that it is in seconds

Reply:
It's already in the config doc, but I guess it doesn't hurt to put it here as well.